### ISSUE TITLE
[Feature] Localization support

### ISSUE THREAD

**Issue by Kvasenok** [2fast-team/2fast#84]
2fast is available only in English. Would be cool to have the app translated into different languages.

**Comment by jp-weber** [comment]
I totally agree with you. I need to see here what would be the most efficient way to have native speakers do translations without being able to program.

**Comment by Kvasenok** [comment]
Usually something like CrowdIn is used for this exact purpose

**Comment by AnxysUaen** [comment]
Extract the text in the view and create a new i18n file.
In order to allow people who do not know programming to participate, csv files are used for translation.
Accept email or PR.

**Comment by AnxysUaen** [comment]
I will submit the first zh_CN.csv

**Comment by jp-weber** [comment]
I have read up a bit here and will probably use transifex in the near future 👍
https://www.transifex.com/open-source/

**Comment by jp-weber** [comment]
Since a subscription is required for a few functions (mobile device management, categories, etc.), I will have the translation translated by AI and make it easily accessible here so that any improvements can be implemented. Free solutions provided by the services by transiflex and crowdin can therefore no longer be used.

**Comment by jp-weber** [comment]
Several languages have now been implemented for some time:
https://github.com/2fast-team/2fast/tree/master/Project2FA.Shared/Strings
